Tasks/AzureTestPlanV0/Invokers/gradleinvoker.ts

```diff
@@ -1,10 +1,9 @@
-import { spawn } from '../testexecutor'
 import tl = require('azure-pipelines-task-lib/task');
 import utils = require('../utils');
 import constants = require('../constants');
 import { execGradleBuild } from '../testLibExecutor';
 
-export async function executegradletests(testsToBeExecuted: string[]) {
+export async function executegradletests(testsToBeExecuted: string[]):Promise<number> {
```

Comment:
minor nit: should the naming be executeGradleTests?. I am seeing that in most places in our code base.

![image](https://github.com/microsoft/azure-pipelines-tasks/assets/100841140/d5e47519-e901-46b0-ba13-d8f83c0214b6)

Reply:
Updated here and for maven and python as well